CMakeLists.txt

```
@@ -112,11 +112,24 @@ endif()
 
 set(LLVM_SPIRV_INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/include)
 
+# first try locating SPIRV-Tools via pkgconfig (the old way)
 pkg_search_module(SPIRV_TOOLS SPIRV-Tools)
-if(NOT SPIRV_TOOLS_FOUND)
-  message(STATUS "SPIRV-Tools not found; project will be built without "
+if (NOT SPIRV_TOOLS_FOUND)
+  # then try locating SPIRV-Tools via cmake (the new way)
+  find_package(SPIRV-Tools)
+  find_package(SPIRV-Tools-tools)
+  if (SPIRV-Tools_FOUND AND SPIRV-Tools-tools_FOUND)
+    set(SPIRV_TOOLS_FOUND TRUE)
+    set(SPIRV_TOOLS_LDFLAGS SPIRV-Tools-shared)
+    get_target_property(SPIRV_TOOLS_INCLUDE_DIRS SPIRV-Tools-shared INTERFACE_INCLUDE_DIRECTORIES)
+  endif()
+endif()
+
+if (NOT SPIRV_TOOLS_FOUND)
```

Shouldn't it be like
```suggestion
if (NOT SPIRV_TOOLS_FOUND AND NOT SPIRV-Tools_FOUND AND NOT SPIRV-Tools-tools_FOUND)
```
?

---

Well, it doesn't change anything in this particular case, but might be good for the safe of consistency